BEST_PRACTICES.md

```diff
@@ -199,7 +199,7 @@ Completion blocks should contain the `error` instead of success/failure blocks.
 
 ## Theming
 
-Use a `Theme.swift` that contains extensions for theming your app. When adding color extensions avoid adding the `Color` postfix to your color methods.
+Use a `Theme.swift` that contains extensions for theming your app. When adding color extensions avoid adding the `Color` postfix to your color methods, also add some context of how this color is used, just be careful to not have one method per UI element.
```

Comment:
Grammar: `just be careful to not have one method per UI element.`
`just be careful not to have one method per UI element.`